plugins/de.ovgu.featureide.fm.core/build.properties

```diff
@@ -5,5 +5,6 @@ bin.includes = META-INF/,\
                src/,\
                schema/,\
                lib/,\
-               plugin.xml
+               plugin.xml,\
+               lib/commons-math-2.2.jar
```

Comment:
Seems to be redundant because of line 7.

Reply:
That's the way of eclipse handling newly added resources